Source/WebCore/platform/mediastream/gstreamer/GStreamerCaptureDeviceManager.h

```diff
@@ -46,8 +47,9 @@ class GStreamerCaptureDeviceManager : public CaptureDeviceManager, public Realti
     ~GStreamerCaptureDeviceManager();
     std::optional<GStreamerCaptureDevice> gstreamerDeviceWithUID(const String&);
 
+    const Vector<CaptureDevice>& speakerDevices();
     const Vector<CaptureDevice>& captureDevices() final;
-    virtual CaptureDevice::DeviceType deviceType() = 0;
+    virtual const OptionSet<CaptureDevice::DeviceType> deviceTypes() const = 0;
```

Comment:
```suggestion
    virtual OptionSet<CaptureDevice::DeviceType> deviceTypes() const = 0;
```

it's better not to return `const T`: see https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-return-const